tools/traffic/flags/flags.go

```diff
@@ -72,6 +72,12 @@ var (
 		Required: false,
 		EnvVar:   common.PrefixEnvVar(envPrefix, "USE_SECURE_GRPC"),
 	}
+	CustomQuorumNumbersFlag = cli.IntSliceFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "custom-quorum-numbers"),
+		Usage:    "Custom quorum numbers to use for the traffic generator",
+		Required: false,
```

Comment:
Does this need a default value (e.g. 0)?

Reply:
default value should be empty